projects/canopy/src/styles/variables/_main.scss

```diff
@@ -49,4 +49,7 @@
 
   /* Separator */
   --separator-color: var(--color-platinum);
+
+  /* z-index */
+  --z-index-1: 1;
```

Comment:
I think we have a z-index section already, may have been merged after your started this work. If you rebase on master, you should see these two:

```
/* z-index */
--z-index-modal-overlay: 1000;
--z-index-modal: 1010;
```

So you could add this new one at the top of this list.